.github/workflows/cloud-tests.yml

```diff
@@ -25,6 +25,27 @@ jobs:
     steps:
       - name: allowed message
         run: echo "Allowed to run"
+      - name: check Github action bot comment
+        if: github.event_name == 'pull_request'
```

Comment:
why this?
we do not want stats if we trigger manually?

Reply:
This is exactly how it was before, if we notice something is missing we can adjust it